.gitignore

```diff
@@ -12,4 +12,5 @@ Thumbs.db
 .idea
 .vscode
 *.sublime-project
-*.sublime-workspace
\ No newline at end of file
+*.sublime-workspace
```

Comment:
I used to think that they should. But in this case you have to explain your opinion to all people who:

1. Commit their editor settings.
2. Commit changes to `.gitignore`.

I gave up 😞